pkg/cache/clusterqueue_snapshot.go

```diff
@@ -88,16 +86,24 @@ func (c *ClusterQueueSnapshot) SimulateUsageRemoval(workloads []*workload.Info)
 }
 
 func (c *ClusterQueueSnapshot) AddUsage(usage workload.Usage) {
-	for fr, q := range usage.Quota {
+	c.AddFRQUsage(usage.Quota)
+	c.updateTASUsage(usage.TAS, add)
+}
+
+func (c *ClusterQueueSnapshot) AddFRQUsage(frq resources.FlavorResourceQuantities) {
```

Comment:
I suppose this is incorrect for TAS? We may need to solve at least part of https://github.com/kubernetes-sigs/kueue/issues/4545

@mimowo

Reply:
yeah, seem incorrect. We need to fix or at least ticket. IIUC https://github.com/kubernetes-sigs/kueue/issues/4545 is just a cleanup, so doesn't belong there imo.

Reply:
https://github.com/kubernetes-sigs/kueue/pull/4603